engine/lite/k8sDeploymentManager/src/test/scala/pl/touk/nussknacker/k8s/manager/PathUtilsTest.scala

```diff
@@ -0,0 +1,25 @@
+package pl.touk.nussknacker.k8s.manager
+
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+import pl.touk.nussknacker.engine.api.process.ProcessName
+import pl.touk.nussknacker.k8s.manager.service.ServicePreparer
+
+class PathUtilsTest extends AnyFunSuite with Matchers {
+
+  private val invalidK8sServiceName = (1 to 70).map(_ => "a").mkString // 63 is the limit
+
+  test("determine default path using scenario name") {
+    PathUtils.defaultPath(ProcessName("foo"), None) shouldEqual "foo"
+
+    val longScenarioNameWithoutNuInstanceIdDefaultPath = PathUtils.defaultPath(ProcessName(invalidK8sServiceName), None)
+    longScenarioNameWithoutNuInstanceIdDefaultPath should contain only 'a'
+    longScenarioNameWithoutNuInstanceIdDefaultPath should have length 63
+
+    val nuInstanceId = (1 to 10).map(_ => "b").mkString
+    val longScenarioNameWithNuInstanceIdDefaultPath = PathUtils.defaultPath(ProcessName(invalidK8sServiceName), Some(nuInstanceId))
+    ServicePreparer.serviceNameWithoutSanitization(Some(nuInstanceId), longScenarioNameWithNuInstanceIdDefaultPath) shouldEqual
+      ServicePreparer.serviceName(Some(nuInstanceId), longScenarioNameWithNuInstanceIdDefaultPath)
+  }
```

Comment:
* Can we have seprate test for each case? The first two are straitghforward, but the third one requires desrptive name IMO.
* Can you please also add test for ProcessName with space?
* What about special characters? I see that some of them are cut, and some replaced? Looks like bug.
```
    PathUtils.defaultPath(ProcessName("a żółć"), None) shouldEqual "a-x"
```

Reply:
1. 2. ok
3. It's look like group of special characters is replaced and after that is added x on the end. Take a look at K8sUtils.sanitizeName - there is plus in first regex (grouping) . I'm not sure if it is bug or intentionally added. I've rather expect one '-' for each replaced character but for me both options are fine. What do you prefer? Should I change it?

Reply:
3. I was also expecting that each character is replaced separatly, but your right - group replacement is also just fine.